src/main/python/3D.py

```diff
@@ -0,0 +1,50 @@
+import plotly
```

Comment:
This file should be a test that does not show anything, except for "pass" or "fail", and we should run it every time the code changes, causing an error e-mail when it changes from "pass" to "fail". This is a bit of work still though.

For now just do not keep test code like this with the main code, but move it to src/test/python